NEWS.md

```diff
@@ -24,6 +24,7 @@ The update introduces breaking changes. If you want to keep using the older vers
 - Users can now supply their own scoring rules to `score()` as a list of functions. Default scoring rules can be accessed using the functions `rules_point()`, `rules_sample()`, `rules_quantile()` and `rules_binary()`, which return a named list of scoring rules suitable for the respective forecast type. Column names of scores in the output of `score()` correspond to the names of the scoring rules (i.e. the names of the functions in the list of scoring rules). 
 - `score()` now returns objects of class `scores` with a stored attribute `score_names` that holds the names of the scoring rules that were used. Users can call `get_score_names()` to access the names of those scoring rules. 
 - `check_forecasts()` was replaced by a different workflow. There now is a function, `as_forecast()`, that determines forecast type of the data, constructs a forecasting object and validates it using the function `validate_forecast()` (a generic that dispatches the correct method based on the forecast type). Objects of class `forecast_binary`, `forecast_point`, `forecast_sample` and `forecast_quantile` have print methods that fulfill the functionality of `check_forecasts()`.
+- Users can test whether an object is of class `forecast_*()` using the function `is_forecast()`.
```

Comment:
```suggestion
- Users can test whether an object is of class `forecast_*()` using the function `is_forecast()`. Users can also test for a specific `forecast_` class using the appropriate `is_forecast.forecast_` method. For example, to check the class of a quantile forecast you would use `as_forecast.forecast_quantile()`.
```

Reply:
Maybe we should add that you'd need to call the internal function, i.e. use `scoringutils:::_is_forecast.forecast_quantile()`

Reply:
oh its internal. Hmm. Yes we definitly need to flag that then or maybe consider exporting if we think this functionality is useful

Reply:
I don't think there is a way to export it (The function currently already has an `export` tag). By default, all methods are internal.

Reply:
ah okay interesting. Lets rile as is then